### PR TITLE
Documentation fixes

### LINKED ISSUE #53: Document features endpoint
See #52 for starting point.

### PR DESCRIPTION
**Pull request by andreasevers**
Fixed #53 & #310
Also fixed some linking issues with `CONTRIBUTING.md`

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@andreasevers Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-commons&pullRequestId=317)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-commons&pullRequestId=317) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@andreasevers Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-commons&pullRequestId=317)!

File: docs/src/main/asciidoc/spring-cloud-commons.adoc [hunk 1]
```diff
@@ -357,7 +357,7 @@ By default, the `ServiceRegistry` implementation will auto-register the running
...
+A `/service-registry` actuator endpoint is provided by Commons. This endpoint relies on a `Registration` bean in the Spring Application Context. Calling `/service-registry` via a GET will return the status of the `Registration`. A POST to the same endpoint with a JSON body will change the status of the current `Registration` to the new value. The JSON body has to include both the `status` and `overridenStatus` field with the preferred value. Please see the documentation of the `ServiceRegistry` implementation you are using for the allowed values for updating the status and the values returned for the status. For instance, Eureka's supported statuses are `UP`, `DOWN`, `OUT_OF_SERVICE` and `UNKNOWN`.
```
Comment (spencergibb) [inline docs/src/main/asciidoc/spring-cloud-commons.adoc]: I don't think this is true:
> body has to include both the `status` and `overridenStatus`

Looking at the code, it should only take `status`.

Comment (andreasevers) [reply docs/src/main/asciidoc/spring-cloud-commons.adoc]: Right, didn't test that combination. Will change this evening!

**Pull request merged**